repl/src/main/resources/fake_shell.py

```diff
@@ -368,6 +369,10 @@ def main():
     sys.stderr = fake_stderr
 
     try:
+        if os.environ.get("LIVY_TEST") == "false":
```

Comment:
Actually, I take my comment back. This won't work at all, because in production `os.environ.get("LIVY_TEST") == "false"` is `False` so you won't run the code inside the condition.

    >>> import os
    >>> os.environ.get("LIVY_TEST") == "false"
    False